wkcuber/api/dataset.py

```diff
@@ -479,16 +507,16 @@ def copy_dataset(
                         Tuple[int, int, int],
                         tuple(
                             BoundingBox(topleft=bbox[0], size=bbox[1])
-                            .align_with_mag(Mag(mag_name), ceil=True)
-                            .in_mag(Mag(mag_name))
+                            .align_with_mag(mag, ceil=True)
+                            .in_mag(mag)
                             .bottomright
                         ),
                     )
                     target_mag.layer.set_bounding_box(offset=bbox[0], size=bbox[1])
 
                     # The data gets written to the target_mag.
                     # Therefore, the chunk size is determined by the target_mag to prevent concurrent writes
-                    mag.get_view().for_zipped_chunks(
+                    mag_view.get_view().for_zipped_chunks(
```

Comment:
I think you are right. The same applies to `target_view=target_mag.get_view()`. The idea behind this is that `mag_view.get_view()` returns the view that is limited to the bounding box from the properties (this works because the default parameter of `get_view()` are used). In general, this makes a difference if the bounding box does not start at `(0, 0, 0)`. However, the function `for_each_chunks` already does this internally for the very same reason. Therefore, this is redundant.